### PR TITLE
Feature/cypres

### PR DESCRIPTION
**Pull request by SanderElias**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: tests/cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (fredericbirke) [inline tests/cypress/fixtures/example.json:1]: This file can be deleted because it's not used?

File: docs/scully-docs/src/main.ts [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
-import { environment } from './environments/environment.js';
+// import { environment } from './src/environments/environment.js';
+// import { AppModule } from './app/app.module.js';
 import { AppModule } from './app/app.module.js';
 
-if (environment.production) {
-  enableProdMode();
-}
+// if (environment.production) {
```
Comment (fredericbirke) [inline docs/scully-docs/src/main.ts:8]: Uncomment or delete

**Review by fredericbirke — CHANGES_REQUESTED** [review]

**Comment by fredericbirke** [comment]
The rest lgtm

File: cypress.config.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      return require('./tests/cypress/plugins/index.js')(on, config);
+    },
+    baseUrl: 'http://localhost:1668',
+    specPattern: 'tests/cypress/integration/**/*.cy.{js,jsx,ts,tsx}',
+    supportFile: 'tests/cypress/support/index.js',
+  },
```
Comment (Jefiozie) [inline cypress.config.ts:16]: I would add the following flag to the setting this way you get a button in Cypress to run all tests after each other.

```suggestion
        experimentalRunAllSpecs: true,
  },
```

**Review by Jefiozie — APPROVED** [review]

File: docs/scully-docs/src/main.ts (hunk 2 above)
Comment (SanderElias) [reply docs/scully-docs/src/main.ts:8]: done

File: cypress.config.ts (hunk 3 above)
Comment (SanderElias) [reply cypress.config.ts:16]: done

**Pull request merged**